File: MT.Model/Dictionary/Position.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region Position:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a Position:EntityBase.
    /// </summary>
    public class Position : EntityBase
    {
        [Key]
        public Guid PositionID { get; set; }
        public string PositionCode { get; set; }
        public string PositionName { get; set; }
        public string Description { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
    }
    #endregion
}

File: BECore/app/model/Position.js
/// <summary>
/// Model EXTJS Position.
/// </summary>
Ext.define('QLDT.model.Position', {
    extend: 'Ext.data.Model',
    idProperty: 'PositionID',
    idType: 'Guid',

    code: 'PositionCode',
    name: 'PositionName',
    fields: [
        { name: "PositionID", type: "string" },
		{ name: "PositionCode", type: "string" },
		{ name: "PositionName", type: "string" },
		{ name: "Description", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});

